@types/interfaces.d.ts

```diff
@@ -4,3 +4,11 @@ export interface AvatarGroupItem {
   img?: string;
   colorIndex?: number;
 }
+
+export interface AvatarI18n {
+  anonymous: string;
+}
+
+export interface AvatarGroupI18n extends AvatarI18n {
+  currentUsers: string;
```

Comment:
Is this correct? I couldn't find `currentUsers` anywhere. 

Shouldn't it be `activeUsers: XXXI18n`, where `XXXI18n` would be an interface with `one: string` and `many: string`?

Reply:
Thanks, done.